tests/qml/helper.cpp

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "../../src/mozillavpn.h"
```

Comment:
empty line between 3 and 4?

Reply:
ok. 

should make a linter for this?. 

i'll clean up everywhere.